irohad/consensus/yac/impl/yac.cpp

```diff
@@ -138,85 +138,101 @@ namespace iroha {
 
       void Yac::closeRound() { timer_->deny(); }
 
+      nonstd::optional<model::Peer> Yac::findPeer(const VoteMessage &vote) {
+        auto peers = cluster_order_.getPeers();
+        auto it =
+            std::find_if(peers.begin(), peers.end(), [&](const auto &peer) {
```

Comment:
capture explicitly vote

Reply:
It is not clear if it will be a better solution here - there are several levels of nesting, leading to quite long capture lists, like `[this, &vote, &answer, &proposal_hash]`.

Reply:
ok, let it left like this